distribution/src/main/packaging/systemd/elasticsearch@.service

```diff
@@ -0,0 +1,60 @@
+[Unit]
+Description=Elasticsearch Instance %i
+Documentation=http://www.elastic.co
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+RuntimeDirectory=elasticsearch
+Environment=ES_HOME=/usr/share/elasticsearch
+Environment=ES_PATH_CONF=${path.conf}/%i
+Environment=PID_DIR=/var/run/elasticsearch
+EnvironmentFile=-${path.env}-%i
+
+WorkingDirectory=/usr/share/elasticsearch
+
+User=elasticsearch
+Group=elasticsearch
+
+ExecStart=/usr/share/elasticsearch/bin/elasticsearch -p ${PID_DIR}/elasticsearch-%i.pid --quiet
+
+# StandardOutput is configured to redirect to journalctl since
+# some error messages may be logged in standard output before
+# elasticsearch logging system is initialized. Elasticsearch
+# stores its logs in /var/log/elasticsearch and does not use
+# journalctl by default. If you also want to enable journalctl
+# logging, you can simply remove the "quiet" option from ExecStart.
+StandardOutput=journal
+StandardError=inherit
+
+# Specifies the maximum file descriptor number that can be opened by this process
+LimitNOFILE=65536
```

Comment:
I don't really like that we have to duplicate (and maintain) all the options like `LimitNOFILE`... Would it be possible to inherit/import from the main service file?